src/roman_datamodels/testing/utils.py

```diff
@@ -701,6 +707,43 @@ def mk_gain(shape=None, filepath=None):
     else:
         return gainref
 
+
+def mk_ipc(shape=None, filepath=None):
+    """
+    Create a dummy IPC instance (or file) with arrays and valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    shape
+        (optional) Shape of array in the model.
+
+    filepath
+        (optional) File name and path to write model to.
+
+    Returns
+    -------
+    roman_datamodels.stnode.IpcRef
+    """
+    meta = {}
+    add_ref_common(meta)
+    ipcref = stnode.IpcRef()
+    meta['reftype'] = 'IPC'
+    ipcref['meta'] = meta
+
+    if not shape:
+        shape = (4096, 4096)
+
+    ipcref['data'] = np.zeros(shape, dtype=np.float32)
```

Comment:
For this dummy case, let's do: [[0, 0, 0], [0, 1, 0], [0, 0, 0]].  This is the trivial no-op IPC kernel.

Reply:
This would break the code were it ever given another shape. Instead I kept the zeros line and added:
`ipcref['data'][int(np.floor(shape[0]/2))][int(np.floor(shape[1]/2))] = 1.0`